pkg/handler/test/helper.go

```diff
@@ -130,7 +130,7 @@ const (
 	// TestServiceAccountHashKey authenticates the service account's token value using HMAC.
 	TestServiceAccountHashKey = "eyJhbGciOiJIUzI1NeyJhbGciOiJIUzI1N"
 	// TestFakeToken signed JWT token with fake data.
```

Comment:
Please extend this comment with the token's expiry

Reply:
done